src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsSignalingSpec.groovy

```diff
@@ -810,22 +888,338 @@ class PriceFloorsSignalingSpec extends PriceFloorsBaseSpec {
         bidResponse.seatbid.first().bid.first().price = requestFloorValue
         bidder.setResponse(ampStoredRequest.id, bidResponse)
 
+        and: "Flush metrics"
+        flushMetrics(floorsPbsService)
+
         when: "PBS processes amp request"
         def response = floorsPbsService.sendAmpRequest(ampRequest)
 
-        then: "PBS should log a errors"
-        assert response.ext?.errors[PREBID]*.code == [999]
-        assert response.ext?.errors[PREBID]*.message ==
-                ["Failed to parse price floors from request, with a reason: " +
-                         "Price floor rules number ${getRuleSize(ampStoredRequest)} " +
-                         "exceeded its maximum number ${MAX_RULES_SIZE}"]
+        then: "PBS should log a warning"
+        def message = "Price floor rules number ${getRuleSize(ampStoredRequest)} " +
+                "exceeded its maximum number ${MAX_RULES_SIZE}"
+        assert response.ext?.warnings[PREBID]*.code == [999]
+        assert response.ext?.warnings[PREBID]*.message == [WARNING_MESSAGE(message)]
+
+        and: "PBS should log a errors"
+        def logs = floorsPbsService.getLogsByTime(startTime)
+        def floorsLogs = getLogsByText(logs, "No price floor data for account $ampRequest.account and " +
+                "request $ampStoredRequest.id, reason: ${WARNING_MESSAGE(message)}")
+        assert floorsLogs.size() == 1
+
+        and: "Alerts.general metrics should be populated"
+        def metrics = floorsPbsService.sendCollectedMetricsRequest()
+        assert metrics[ALERT_GENERAL] == 1
 
         where:
         maxRules       | maxRulesSnakeCase
         MAX_RULES_SIZE | null
         null           | MAX_RULES_SIZE
     }
 
+    def "PBS shouldn't emit error in log and response when data is invalid and floors status is in progress"() {
+        given: "Default BidRequest with empty floors.data"
+        def bidRequest = bidRequestWithFloors.tap {
+            ext.prebid.floors.data = null
+        }
+
+        and: "Account with invalid floors config"
+        def account = getAccountWithEnabledFetch(bidRequest.accountId).tap {
+            config.auction.priceFloors.fetch.enabled = true
+        }
```

Comment:
Why invalid?
And `getAccountWithEnabledFetch` is creating to `fetch.enabled = true`